blob/s3blob/s3blob.go

```diff
@@ -670,12 +667,17 @@ func (b *bucket) NewTypedWriter(ctx context.Context, key string, contentType str
 	}, nil
 }
 
-var errUnimplemented = errors.New("not implemented")
-
 // Copy implements driver.Copy.
 func (b *bucket) Copy(ctx context.Context, dstKey, srcKey string, opts *driver.CopyOptions) error {
-	// TODO(rvangent): Implement this and delete errUnimplemented.
-	return errUnimplemented
+	dstKey = escapeKey(dstKey)
+	srcKey = escapeKey(srcKey)
+	input := &s3.CopyObjectInput{
+		Bucket:     aws.String(b.name),
+		CopySource: aws.String(srcKey),
+		Key:        aws.String(dstKey),
+	}
+	_, err := b.client.CopyObject(input)
```

Comment:
Can you change this to `CopyObjectWithContext(ctx, input)` so it respect the ctx? I notice that there are other requests in this file that don't do this, I'll send a separate PR to fix.

Reply:
I did send a separate PR already for this, so updating this line should make it consistent with the rest.